cmd/storage.go

```diff
@@ -71,12 +77,24 @@ var storageUnmountCmd = &cobra.Command{
 		projectName := project.GetCurrent(client)
 		componentName := getComponent(client, storageComponent, applicationName, projectName)
 
-		storageName := args[0]
-		exists, err := storage.IsMounted(client, storageName, componentName, applicationName)
-		checkError(err, "")
-		if !exists {
-			fmt.Printf("Storage %v does not exist in component %v\n", storageName, componentName)
-			os.Exit(1)
+		storageName := ""
+		// checking if the first character in the argument is a "/", indicating a path or not, indicating a storage name
+		if string(args[0][0]) == "/" {
+			path := args[0]
+			storageName, err = storage.GetStorageNameFromMountPath(client, path, componentName, applicationName)
+			checkError(err, "")
```

Comment:
instead of `""` maybe add explanation (`Unable to get storage name from mount path`) or something similar?